lua/compe_emoji/init.lua

```diff
@@ -0,0 +1,29 @@
+local compe = require'compe'
+
+local Source = {}
+
+Source.get_metadata = function(_)
+  return {
+    priority = 80,
+    dup = 1,
+    menu = '[Emoji]'
+  }
+end
+
+Source.determine = function(_, context)
+  return compe.helper.determine(context, {
+    keyword_pattern = [[:\w*$]]
```

Comment:
I think the source should be triggerd only when the line is `\%(\s\|^\)\zs:\w*`.

Reply:
For example, the PHP/Rust has `Identifier::CONST` syntax.